Seneca-CDOT/telescope
# Replace ThemeToggleButton with ThemeSelector

## Issue This PR Addresses
This PR addresses potentially fixes #3719 that provides additional themes for users to select. I have added `Light High Contrast` and `Dark Dim` colour themes in this PR. 



## Type of Change




## Description
I have added `ThemeSelector` components as requested in issue #3719. I have removed toggle button and add popover selector to change the theme. 

![Screenshot 2022-11-16 at 10 05 31 AM](https://user-images.githubusercontent.com/59734889/202216987-95872c8a-23a6-457d-a034-596bafd461d5.png)

There might be an update needed to support `GitHubContributionCard` but I don't see any part that imports `GitHubContributionCard` module. Please let me know if I miss anything. 



## Steps to test the PR
1. Scroll down on main page until the sidebar menu is displayed
2. Click the last palette icon
3. Select the theme to see if it's working
No special setup is required



## Checklist
I think this PR needs a UI test. But I don't see UI test set up in this project. Please let me know if any test needs to be written for my PR.
